src/Storages/ObjectStorageQueue/ObjectStorageQueueUnorderedFileMetadata.cpp

```diff
@@ -141,19 +131,13 @@ void ObjectStorageQueueUnorderedFileMetadata::setProcessedImpl(bool remove_proce
         request_index[CHECK_PROCESSING_ID_PATH] = 0;
         request_index[REMOVE_PROCESSING_ID_PATH] = 1;
         request_index[REMOVE_PROCESSING_PATH] = 2;
-
-        if (!remove_processing_nodes_only)
-            request_index[SET_PROCESSED_PATH] = 3;
-    }
-    else if (!remove_processing_nodes_only)
-    {
-        request_index[SET_PROCESSED_PATH] = 0;
+        request_index[SET_PROCESSED_PATH] = 3;
```

Comment:
Why you set it here if you will set it to something else right after :confused:

Reply:
that was accidental - removed `remove_processing_nodes_only` too quickly without properly looking